### PR TITLE
DDCE-3603 - added scalafmt, scalastyle, updated deps, updated sbt, fi…

### PR DESCRIPTION
**Pull request by frikit**
…x all warnings

### CODE REVIEW

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala [hunk 1]
```diff
@@ -16,73 +16,90 @@
...
     val _ :: product :: status :: expired =
       makeString.toUpperCase.replace("--", "_").split("-").toList
 
-    val now = LocalDate.now()
-    val isExpired = expired.contains("EX")
+    val now         = LocalDate.now()
+    val isExpired   = expired.contains("EX")
     val expiredDate = Some(if (isExpired) now.minusDays(1) else now.plusDays(1))
     StatusCheckResult(
       "Michael Makson",
```
Comment (lmugnai) [inline app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala:86]: For test data in the open - please follow this guidance : https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/DemoStubData.scala [hunk 2]
```diff
@@ -22,7 +22,7 @@ import java.time.LocalDate
 
 object DemoStubData extends DataSet {
 
-  val lawrenceVelazquez = StatusCheckResult(
+  val lawrenceVelazquez: StatusCheckResult = StatusCheckResult(
```
Comment (lmugnai) [inline app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/DemoStubData.scala:25]: All these names should really be changed to follow the correct test data in the open guidance : https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open

Happy to create a separate ticket for this if you want?

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 3]
```diff
@@ -34,88 +50,94 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
       ping.status.shouldBe(200)
 
       val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}"""
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:53]: lots of names in here too which need updating

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/models/ImmigrationStatus.scala [hunk 4]
```diff
@@ -16,9 +16,10 @@
 
 package uk.gov.hmrc.homeofficesettledstatusstubs.models
 
-import java.time.LocalDate
 import play.api.libs.json.{Format, Json}
 
+import java.time.LocalDate
+
 final case class ImmigrationStatus(
   // start date of this status
```
[the remark this answers is not in the document]
Comment (frikit) [reply app/uk/gov/hmrc/homeofficesettledstatusstubs/models/ImmigrationStatus.scala]: done

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/models/StatusResponse.scala [hunk 5]
```diff
@@ -19,12 +19,13 @@ package uk.gov.hmrc.homeofficesettledstatusstubs.models
 import play.api.data.FormError
 import play.api.http.Status.BAD_REQUEST
 import play.api.libs.json.{Json, OWrites}
+import play.shaded.ahc.io.netty.handler.codec.http.HttpResponseStatus
```
[the remark this answers is not in the document]
Comment (frikit) [reply app/uk/gov/hmrc/homeofficesettledstatusstubs/models/StatusResponse.scala]: done

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala (hunk 1 above)
Comment (frikit) [reply app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala]: done

File: app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/DemoStubData.scala (hunk 2 above)
Comment (frikit) [reply app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/DemoStubData.scala]: done

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala (hunk 3 above)
Comment (frikit) [reply it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala]: done

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 6]
```diff
@@ -1,7 +1,24 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package uk.gov.hmrc.homeofficesettledstatusstubs.endpoints
 
 import org.scalatest.Suite
 import org.scalatestplus.play.ServerProvider
+import play.api.http.Status
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:21]:
```suggestion
import play.api.http.Status._
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 7]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
   "POST /v1/status/public-funds/mrz" should {
 
     "respond with 200 if request is valid" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:48]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 8]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
+      )
 
-      (result.json.as[JsObject] \ "result").get shouldBe Json.toJson(DemoStubData.lawrenceVelazquez)
+      (result.json.as[JsObject] \ "result").get shouldBe Json.toJson(DemoStubData.testData1)
 
     }
 
     "respond with 404 if the service failed to find an identity because of incorrect number" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:59]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 9]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = publicFundsByMRZ(
-        s"""{"documentType":"PASSPORT", "documentNumber" : "NOT VALID", "nationality": "CHE","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"documentType":"PASSPORT", "documentNumber" : "NOT VALID", "nationality": "CHE","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 404
+      result.status          shouldBe Status.NOT_FOUND
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:65]:
```suggestion
      result.status          shouldBe NOT_FOUND
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 10]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
-        haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
-      ))
+          "error",
+          haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
+        ))
     }
 
     "respond with 404 if the service failed to find an identity because of a nationality" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:74]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 11]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = publicFundsByMRZ(
-        s"""{"documentType":"PASSPORT", "documentNumber" : "NOT VALID", "nationality": "INVALID","dateOfBirth":"1954-10-04", $statusRange}""")
-
+        s"""{"documentType":"PASSPORT", "documentNumber" : "NOT VALID", "nationality": "INVALID","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 404
+      result.status          shouldBe Status.NOT_FOUND
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:80]:
```suggestion
      result.status          shouldBe NOT_FOUND
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 12]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
-        haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
-      ))
+          "error",
+          haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
+        ))
     }
 
     "respond with 400 if one of the required input parameters is missing from the request" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:89]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 13]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = publicFundsByMRZ(
-        s"""{"documentNumber" : "NOT VALID", "nationality": "INVALID","dateOfBirth":"1954-10-04", $statusRange}""")
-
+        s"""{"documentNumber" : "NOT VALID", "nationality": "INVALID","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 400
+      result.status          shouldBe Status.BAD_REQUEST
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:95]:
```suggestion
      result.status          shouldBe BAD_REQUEST
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 14]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
-
     "respond with 200 if statuses empty" in {
       val result = publicFundsByMRZ(
-        s"""{"documentType":"NAT", "documentNumber" : "ZL341566", "nationality": "ESP", "dateOfBirth":"1956-10-09", $statusRange}""")
+        s"""{"documentType":"NAT", "documentNumber" : "ZL341566", "nationality": "ESP", "dateOfBirth":"1956-10-09", $statusRange}"""
+      )
 
-      result.status shouldBe 200
+      result.status shouldBe Status.OK
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:108]:
```suggestion
      result.status shouldBe OK
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 15]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 409 if status is 409" in {
       val result = publicFundsByMRZ(
         s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB3", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}"""
       )
 
-      result.status shouldBe 409
+      result.status shouldBe Status.CONFLICT
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:116]:
```suggestion
      result.status shouldBe CONFLICT
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 16]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 429 if status is 429" in {
       val result = publicFundsByMRZ(
-        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB4", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}""")
+        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB4", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}"""
+      )
 
-      result.status shouldBe 429
+      result.status shouldBe Status.TOO_MANY_REQUESTS
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:124]:
```suggestion
      result.status shouldBe TOO_MANY_REQUESTS
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala [hunk 17]
```diff
@@ -28,88 +45,92 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 500 if status is 500" in {
       val result = publicFundsByMRZ(
-        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB5", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}""")
+        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB5", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}"""
+      )
 
-      result.status shouldBe 500
+      result.status shouldBe Status.INTERNAL_SERVER_ERROR
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala:132]:
```suggestion
      result.status shouldBe INTERNAL_SERVER_ERROR
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 18]
```diff
@@ -1,7 +1,24 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package uk.gov.hmrc.homeofficesettledstatusstubs.endpoints
 
 import org.scalatest.Suite
 import org.scalatestplus.play.ServerProvider
+import play.api.http.Status
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:21]:
```suggestion
import play.api.http.Status._
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 19]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
   "POST /v1/status/public-funds/nino" should {
 
     "respond with 200 if request is valid" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:51]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 20]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"$NINO_VALID","givenName":"John","familyName":"Doe","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 200
-      (result.json.as[JsObject] \ "result").toOption shouldBe Some(Json.toJson(DemoStubData.lawrenceVelazquez))
+      result.status                                  shouldBe Status.OK
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:57]:
```suggestion
      result.status                                  shouldBe OK
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 21]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
+      (result.json.as[JsObject] \ "result").toOption shouldBe Some(Json.toJson(DemoStubData.testData1))
     }
 
     "respond with 200 if request is valid, and date contains a pattern, and nino has no suffix" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:62]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 22]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"BR253184", "givenName":"Njeri", "familyName":"Samara", "dateOfBirth":"1973-XX-16", $statusRange}""")
+        s"""{"nino":"BR253184", "givenName":"John", "familyName":"Doe2", "dateOfBirth":"1973-XX-16", $statusRange}"""
+      )
 
-      result.status shouldBe 200
-      (result.json.as[JsObject] \ "result").get shouldBe Json.toJson(DemoStubData.njeriSamara)
+      result.status                             shouldBe Status.OK
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:68]:
```suggestion
      result.status                             shouldBe OK
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 23]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
+      (result.json.as[JsObject] \ "result").get shouldBe Json.toJson(DemoStubData.testData2)
     }
 
     "respond with 404 if the service failed to find an identity because of nino" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:73]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 24]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID_BUT_UNKNOWN","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"$NINO_VALID_BUT_UNKNOWN","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 404
+      result.status          shouldBe Status.NOT_FOUND
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:79]:
```suggestion
      result.status          shouldBe NOT_FOUND
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 25]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
-        haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
-      ))
+          "error",
+          haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
+        ))
     }
 
     "respond with 404 if the service failed to find an identity because of a given name" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:88]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 26]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID","givenName":"Bawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"$NINO_VALID","givenName":"Bawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 404
+      result.status          shouldBe Status.NOT_FOUND
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:94]:
```suggestion
      result.status          shouldBe NOT_FOUND
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 27]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
-        haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
-      ))
+          "error",
+          haveProperty[String]("errCode", be("ERR_NOT_FOUND"))
+        ))
     }
 
     "respond with 400 if one of the required input parameters is missing from the request" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:103]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 28]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
-      val result = callEndpoint(
-        s"""{"nino":"$NINO_VALID","givenName":"Lawrence","familyName":"Velazques", $statusRange}""")
+      val result =
+        callEndpoint(s"""{"nino":"$NINO_VALID","givenName":"John","familyName":"Doe", $statusRange}""")
 
-      result.status shouldBe 400
+      result.status          shouldBe Status.BAD_REQUEST
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:108]:
```suggestion
      result.status          shouldBe BAD_REQUEST
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 29]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
-        haveProperty[String]("errCode", be("ERR_VALIDATION"))
-      ))
+          "error",
+          haveProperty[String]("errCode", be("ERR_VALIDATION"))
+        ))
     }
 
     "respond with 400 if one of the input parameters passed in has failed validation" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:117]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 30]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"invalid","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}""")
+        s"""{"nino":"invalid","givenName":"Lawrence","familyName":"Velazques","dateOfBirth":"1954-10-04", $statusRange}"""
+      )
 
-      result.status shouldBe 400
+      result.status          shouldBe Status.BAD_REQUEST
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:123]:
```suggestion
      result.status          shouldBe BAD_REQUEST
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 31]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
+              "fields",
+              haveProperty[String]("name", be("nino"))
+                and haveProperty[String]("code", be("ERR_INVALID_NINO"))
+            )
+        ))
     }
 
     "respond with 200 if statuses empty" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:137]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 32]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"ZL341566D","givenName":"N","familyName":"Sabina","dateOfBirth":"1956-10-09", $statusRange}""")
+        s"""{"nino":"ZL341566D","givenName":"N","familyName":"Sabina","dateOfBirth":"1956-10-09", $statusRange}"""
+      )
 
-      result.status shouldBe 200
+      result.status shouldBe Status.OK
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:143]:
```suggestion
      result.status shouldBe OK
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 33]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 429 if status is 429" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:147]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 34]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"TP991941C","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}""")
+        s"""{"nino":"TP991941C","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}"""
+      )
 
-      result.status shouldBe 429
+      result.status shouldBe Status.TOO_MANY_REQUESTS
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:153]:
```suggestion
      result.status shouldBe TOO_MANY_REQUESTS
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 35]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 409 if status is 409" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:157]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 36]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"HK089820A","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}""")
+        s"""{"nino":"HK089820A","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}"""
+      )
 
-      result.status shouldBe 409
+      result.status shouldBe Status.CONFLICT
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:163]:
```suggestion
      result.status shouldBe CONFLICT
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 37]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 500 if status is 500" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:167]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala [hunk 38]
```diff
@@ -31,121 +48,130 @@ class NinoSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
 
       val result = callEndpoint(
-        s"""{"nino":"BY880209A","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}""")
+        s"""{"nino":"BY880209A","givenName":"J","familyName":"Does","dateOfBirth":"2001-XX-31", $statusRange}"""
+      )
 
-      result.status shouldBe 500
+      result.status shouldBe Status.INTERNAL_SERVER_ERROR
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/NinoSearchEndpointISpec.scala:173]:
```suggestion
      result.status shouldBe INTERNAL_SERVER_ERROR
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala [hunk 39]
```diff
@@ -1,7 +1,24 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package uk.gov.hmrc.homeofficesettledstatusstubs.endpoints
 
 import org.scalatest.Suite
 import org.scalatestplus.play.ServerProvider
+import play.api.http.Status
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala:21]:
```suggestion
import play.api.http.Status._
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala [hunk 40]
```diff
@@ -32,22 +49,22 @@ class TokenEndpointISpec extends ServerBaseISpec with JsonMatchers {
   "POST /v1/status/public-funds/token" should {
 
     "respond with 200 and return a token when valid request" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala:52]:
```suggestion
      ping.status.shouldBe(OK)
```

File: it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala [hunk 41]
```diff
@@ -32,22 +49,22 @@ class TokenEndpointISpec extends ServerBaseISpec with JsonMatchers {
...
     }
 
     "respond with 400 when invalid request" in {
-      ping.status.shouldBe(200)
+      ping.status.shouldBe(Status.OK)
 
       val result = token("client_credentials", "invalid", "TBC")
 
-      result.status shouldBe 400
+      result.status shouldBe Status.BAD_REQUEST
```
Comment (lmugnai) [inline it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/TokenEndpointISpec.scala:67]:
```suggestion
      result.status shouldBe BAD_REQUEST
```

**Review by lmugnai — COMMENTED** [review]
Can you merge these suggestions please?

**Review by lmugnai — APPROVED** [review]

**Pull request merged**